pkg/dataplane/util/ansible_execution.go

```diff
@@ -225,6 +226,15 @@ func AnsibleExecution(
 		ansibleEE.Spec.ExtraMounts = append(aeeSpec.ExtraMounts, []storage.VolMounts{ansibleEEMounts}...)
 		ansibleEE.Spec.Env = aeeSpec.Env
 
+		// Validate volume names
+		for _, emount := range ansibleEE.Spec.ExtraMounts {
+			for _, vol := range emount.Volumes {
+				if msgs := apimachineryvalidation.IsDNS1123Label(vol.Name); len(msgs) != 0 {
+					return fmt.Errorf("volume name %q is not a valid: %s", vol.Name, strings.Join(msgs, ";"))
+				}
+			}
+		}
```

Comment:
We should validate this with the webhook right? Rather than accepting values that we can't work with and then raising errors during the reconcile process?

Reply:
just moved it to the webhook